tests/system/feature_store/test_feature_store.py

```diff
@@ -4202,6 +4202,36 @@ def test_get_online_features_after_ingest_without_inference(self):
                 f"store://feature-vectors/{self.project_name}/my-vector:latest"
             )
 
+    def test_ingest_with_rename_columns(self):
+        csv_path = str(self.assets_path / "fields_with_space.csv")
+        run_id = str(uuid.uuid4())
+        name = f"test_ingest_with_rename_columns_{run_id}"
+        data = pd.read_csv(csv_path)
+        expected_result = data.copy().rename(
+            columns={"city of birth": "city_of_birth"}
+        )[["city_of_birth"]]
```

Comment:
I'd make it
```suggestion
        )[["name", "city_of_birth"]]
        expected_result.set_index("name", inplace=True)
```

Reply:
Nice! I did something very similar:

`expected_result = data.copy().rename(
            columns={"city of birth": "city_of_birth"}
        ).set_index('name')`